server/keyshare/tasks/tasks_test.go

```diff
@@ -35,6 +37,21 @@ func countRows(t *testing.T, db *sql.DB, table, where string) int {
 	return count
 }
 
+func getDeleteDates(t *testing.T, db *sql.DB) map[string]int {
```

Comment:
In Go, it is uncommon to have a `get` prefix for a getter. This is not a textbook example of a getter of course, but maybe nice to follow that convention here too.

https://go.dev/doc/effective_go#Getters

Reply:
Thanks, old habits die hard. I renamed it to `countDeleteDates` since `deleteDates` would imply deleting dates.